sdk/src/types.rs

```diff
@@ -862,4 +980,65 @@ mod tests {
         // Test invalid private key hex literal.
         assert!(LocalAccount::from_private_key("invalid_private_key", 0).is_err());
     }
+
+    #[ignore]
+    #[tokio::test]
+    async fn test_derive_keyless_account() {
+        let aptos_rest_client = Client::builder(AptosBaseUrl::Devnet).build();
+        let jwt = "eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCIsImtpZCI6InRlc3QtcnNhIn0.eyJpc3MiOiJ0ZXN0Lm9pZGMucHJvdmlkZXIiLCJhdWQiOiJ0ZXN0LWtleWxlc3MtZGFwcCIsInN1YiI6InRlc3QtdXNlci0wIiwiZW1haWwiOiJ0ZXN0QGFwdG9zbGFicy5jb20iLCJlbWFpbF92ZXJpZmllZCI6dHJ1ZSwiaWF0IjoxNzI1NDc1MTEyLCJleHAiOjI3MDAwMDAwMDAsIm5vbmNlIjoiNzA5NTI0MjMzMzk2NDQ1NzI2NzkzNDcyMzc2ODA4MDMwMzMyNDQ2MjgyMTE5MTc1NjQwOTQ1MDA5OTUxOTc4MTA1MTkxMDE4NzExOCJ9.eHqJLdje0FRD3UPmSw8sFHRYe9lwqSydAMcfHcpxkFwew2OTy6bWFsLQTdJp-eCZPhNzlfBXwNxaAJZksCWFWkzCz2913a5b88XRT9Im7JBDtA1e1IBXrnfXG0MDpsVRAuRNzLWqDi_4Fl1OELvoEOK-Tl4cmIwOhBr943S-b14PRVhrQ1XBD5MXaHWcJyxMaEtZfu_xxCQ-jjR---iguD243Ze98JlcOIV8VmEBg3YiSyVdMDZ8cgRia0DI8DwFn7rIxaV2H5FXb9JcehLgNP82-gsfEGV0iAXuBk7ZvRzMVA-srE9JvxVOyq5UkYu0Ss9LjKzX0KVojl7Au_OxGA";
```

Comment:
Please don't write tests like this: no one can see what is in that JWT. 

If this test fails months down the road, it's gonna be hard to debug.

Either:
1. Use human-readable, test JWTs from our codebase. That's what I used in the authenticator.
2. If you need to hardcode a specific JWT, at least hardcode the plaintext JSON and call `base64url_encode` on it to derive the JWT.

Reply:
I agree.  But this isn't actually supposed to test (it is ignored).  Rather it's just meant to be an example of usage.

But will do number 2.

Reply:
I left a comment explaining instead.